waiter/src/waiter/core.clj

```diff
@@ -1485,6 +1490,22 @@
                              (wrap-secure-request-fn
                                (fn waiter-auth-handler-fn [request]
                                  {:body (str (:authorization/user request)), :status 200})))
+   :waiter-auth-saml-acs-handler-fn (pc/fnk [[:state authenticator]]
+                                      (let [{:keys [saml-acs-handler-fn]} authenticator]
+                                        (fn waiter-auth-saml-acs-handler-fn [request]
+                                          (when-not saml-acs-handler-fn
```

Comment:
I don't think `saml-acs-handler-fn` should come from the authenticator via field access. Why not refer to it directly?

Reply:
done